src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <a href="#" class="star star-off"></a>
+      <a href="#" class="star star-off"></a>
+      <a href="#" class="star star-off"></a>
+      <a href="#" class="star star-off"></a>
+      <a href="#" class="star star-off"></a>
+    </div>
+
+    <div class="stars stars--1">
+      <a href="#" class="star star-on"></a>
+      <a href="#" class="star star-off"></a>
+      <a href="#" class="star star-off"></a>
+      <a href="#" class="star star-off"></a>
+      <a href="#" class="star star-off"></a>
+    </div>
+
+    <div class="stars stars--2">
+      <a href="#" class="star star-on"></a>
+      <a href="#" class="star star-on"></a>
+      <a href="#" class="star star-off"></a>
+      <a href="#" class="star star-off"></a>
+      <a href="#" class="star star-off"></a>
+    </div>
+
+    <div class="stars stars--3">
+      <a href="#" class="star star-on"></a>
+      <a href="#" class="star star-on"></a>
+      <a href="#" class="star star-on"></a>
+      <a href="#" class="star star-off"></a>
+      <a href="#" class="star star-off"></a>
+    </div>
```

Comment:
```suggestion
    <div class="stars stars--3">
      <a href="#" class="stars__star"></a>
      <a href="#" class="stars__star"></a>
      <a href="#" class="stars__star"></a>
      <a href="#" class="stars__star"></a>
      <a href="#" class="stars__star"></a>
    </div>
```
implement this task with following html